cf-agent/verify_users_pam.c

```diff
@@ -903,8 +903,9 @@ static bool SetAccountLocked(const char *puser, const char *hash, bool lock)
 
     return SetAccountLockExpiration(puser, lock);
 }
+static void TransformGidsToGroups(StringSet **list);
 
-static bool GroupGetUserMembership (const char *user, StringSet *result)
+static bool GetGroupInfo (const char *user, const User *u, StringSet **groups_to_set, StringSet **groups_missing, StringSet **current_secondary_groups)
 {
```

Comment:
changed name because we needed to get more different bits of information not just membership.